inbm/packaging/windows-override/install.ps1

```diff
@@ -35,13 +35,29 @@ if ($env:UCC_MODE) {
     "TRUE" | Set-Content -Path $UCC_FILE
 }
 
-
+# This loop iterates through a list of folders, creates them if they don't exist,  
+# sets their Access Control List (ACL) protections, and adds two access rules, granting FullControl 
+# privileges to SYSTEM and Administrators groups on the specified folders.

```

Comment:
Just to confirm:
1. Administrators and SYSTEM will have full access to the files (secrets/certs/keys)
2. INBM services will also have access to the these (their respective) files
3. UCC agent will have access to its set of certs/keys etc.
4. non-admin or non-INBM/UCC service will not have access to these folders/files.

Reply:
@pillaiabhishek Currently I'm expecting all services to run as SYSTEM (as that's the convention in Windows), so only SYSTEM/Administrators will have any access to any of those files.

Reply:
@pillaiabhishek and correct, standard users will not have access to anything they could use to cause trouble; and the INBM/UCC services will have the access they need.

Reply:
ok. Please let UCC team know that their agent would also need to run as SYSTEM.